sw/device/silicon_creator/lib/cert/BUILD

```diff
@@ -112,6 +112,42 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "cbor_helper",
+    hdrs = ["cbor_helper.h"],
+    deps = [
+        "@open-dice//:cbor_reader_writer",
+        "//sw/device/lib/base:status",
+        "//sw/device/lib/runtime:log",
+    ],
+)
+
+cc_library(
+    name = "dice_cwt",
+    srcs = ["dice_cwt.c"],
+    hdrs = ["dice.h"],
+    deps = [
+        ":cert",
+        ":cbor_helper",
+        "//hw/ip/otp_ctrl/data:otp_ctrl_c_regs",
+        "//sw/device/lib/base:memory",
+        "//sw/device/lib/base:status",
+        "//sw/device/lib/testing/test_framework:check",
+        "//sw/device/silicon_creator/lib:attestation",
+        "//sw/device/silicon_creator/lib:otbn_boot_services",
+        "//sw/device/silicon_creator/lib/base:util",
+        "//sw/device/silicon_creator/lib/cert:cdi_0_template_library",
+        "//sw/device/silicon_creator/lib/cert:cdi_1_template_library",
+        "//sw/device/silicon_creator/lib/cert:uds_template_library",
+        "//sw/device/silicon_creator/lib/drivers:hmac",
+        "//sw/device/silicon_creator/lib/drivers:keymgr",
+        "//sw/device/silicon_creator/lib/drivers:lifecycle",
+        "//sw/device/silicon_creator/lib/drivers:otp",
+        "//sw/device/silicon_creator/lib/sigverify:ecdsa_p256_key",
+        "//sw/device/silicon_creator/manuf/lib:flash_info_fields",
```

Comment:
can you double check the deps you have here, I don't think you need a lot of these